libs/remix-ui/solidity-compiler/src/lib/css/style.css

```diff
@@ -154,6 +154,11 @@
 .remixui_iconbtn {
   margin-right: 0.3em;
 }
+.remixui_btnBlock {
+  display: block;
+  width: 100%;
+  word-wrap: break-word;
```

Comment:
Please use Boostrap classes when possible
w-100
text-break

Reply:
sweeet, thanks for that, i just pushed the changes :)

Reply:
@LianaHus seems like the ci pipeline failed again due to timeout, let me know if there are any other changes you need